server/webserver.go

```diff
@@ -12,6 +12,8 @@ import (
 	"strings"
 	"time"
 
+	_ "net/http/pprof"
```

Comment:
Lint gives this warning:
`a blank import should be only in a main or test package, or have a comment justifying it`
Can we add comment here to satisfy linter?

Reply:
Curious, what linter / command shows this message? I'm not seeing it with `golint cmd/livepeer/livepeer.go` .

FWIW, this line is basically a copy-paste from https://golang.org/pkg/net/http/pprof/ . But agreed it'd be good to have an explanatory comment here; just want to make sure I'm satisfying the linter properly.

Reply:
`golint server/webserver.go`

It is ran by default by VSCode.

It is satisfied by any comment.

Reply:
Doh, ran it `golint` against the wrong file. Fixed.